### ISSUE TITLE
It doesn't work on Intraday ms files.

### ISSUE THREAD

**Issue by nabil-ha** [themech/ms2txt#6]
Will it support intraday any soon?

**Comment by themech** [comment]
It supports only EOD data because my trading platform provided EOD in this format. Intraday files are available in text format.

Anyway, I'm ok with adding the support for intraday pretty soon, I would just need to access to some intraday files in Metastock format so I can check them out and test while making changes. I'll get back with some more info soon.

**Comment by nabil25x** [comment]
Give me your email, I'll send you some

**Comment by themech** [comment]
That would be very helpful, my email is mech@themech.net. Thank you

**Comment by themech** [comment]
@nabil25x It looks like the only difference is that intraday files have the extra time column just after the date one. I did a small code change and it looks like it is working for me (also I used `ms2txt.py -l -e cp1256` for Arabic files). Can you please use the newest version and verify if it works fine for you?

**Comment by nabil-ha** [comment]
Sure I'm going to try it.
But excuse me this period, I have exams.
Thanks

**Comment by johnbest5673** [comment]
Will it convert Daily data to Weekly or Monthly?

**Comment by themech** [comment]
@johnbest5673 no, it simply extracts the binary formal. But once you have text files, it is pretty easy to convert intra files to daily or monthly, few lines of Python or maybe even a spreadsheet should do a trick
